Foundation/NSURLProtocol.swift

```diff
@@ -471,7 +473,7 @@ open class URLProtocol : NSObject {
     }
 
     internal class func getProtocols() -> [AnyClass]? {
-        return _registeredProtocolClasses
+        return _nativeProtocolClasses
```

Comment:
This is going to break anyone who registers their protocol classes via the supported APIs, right? Creating a special-case for HTTPURLProtocol does not look like the way to fix this problem. Instead, the protocol classes should be added using a dispatch_once style mechanism, so that they aren't added each time.